client/my-sites/plans/jetpack-plans/product-lightbox/index.tsx

```diff
@@ -32,7 +32,7 @@ const TagItems: React.FC< { tags: JetpackTag[] } > = ( { tags } ) => (
 	<>
 		{ tags.map( ( tag ) => (
 			<div className="product-lightbox__detail-tags-tag" key={ tag.tag }>
-				<span>{ Tags[ tag.tag ] }</span>
+				<span aria-hidden="true">{ Tags[ tag.tag ] }</span>
```

Comment:
Making sure icons are not read by screen readers here, since they are purely decorative.